app/components/new-project-modal/component.js

```diff
@@ -0,0 +1,74 @@
+import { computed } from '@ember/object';
+import { A } from '@ember/array';
+import { inject as service } from '@ember/service';
+import Component from '@ember/component';
+import { task, timeout } from 'ember-concurrency';
+
+export default Component.extend({
+    store: service(),
+    newNode: null,
+    more: false,
+    institutionsSelected: computed.oneWay('user.institutions'),
+
+    actions: {
+        select(institution) {
+            this.set('institutionsSelected', this.get('institutionsSelected').slice());
+            if (this.get('institutionsSelected').includes(institution)) {
+                this.get('institutionsSelected').removeObject(institution);
+            } else {
+                this.get('institutionsSelected').pushObject(institution);
+            }
+            this.notifyPropertyChange('institutionsSelected');
+        },
+        selectAll() {
+            this.set('institutionsSelected', this.get('user.institutions').slice());
+        },
+        removeAll() {
+            this.set('institutionsSelected', A([]));
+        },
+        toggleMore() {
+            this.toggleProperty('more');
+        },
+        closeModal() {
+            this.set('newNode', null);
+            this.get('closeModal')();
+        },
+        reload() {
+            this.set('newNode', null);
+            this.get('closeModal')();
+            this.get('reloadNodes')();
+        },
+        pushEnter() {
+            if (this.get('nodeTitle.length')) {
+                this.get('createNode').perform();
+            }
+        },
+    },
+
+    findNodes: task(function* (term) {
+        yield timeout(500);
+        const user = yield this.get('user');
+        const nodes = yield user.queryHasMany('nodes', { 'filter[title]': term });
```

Comment:
Just FYI, you can pass query params with [] as nested objects, like `{ filter: { title: term } }`. Either way's fine, but the object is nice if you're filtering several things.